### PR TITLE
fix(TDP-4926) : fix on action part only (not pipeline)

### PR DESCRIPTION
**Pull request by tlnd-jbrun**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4629
https://jira.talendforge.org/browse/TDP-5057

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/125/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/125/)

File: dataprep-actions/src/main/java/org/talend/dataprep/transformation/actions/common/ActionsUtils.java [hunk 1]
```diff
@@ -76,14 +76,17 @@ public static void createNewColumn(ActionContext context, List<AdditionalColumn>
...
+                if (additionalColumn.getCopyMetadataFromId() != null) {
+                    ColumnMetadata newColumn = context.getRowMetadata().getById(additionalColumn.getCopyMetadataFromId());
+                    brandNewColumnBuilder.copy(newColumn).computedId(StringUtils.EMPTY);
+                    brandNewColumnBuilder.type(Type.get(newColumn.getType()));
+                } else {
+                    brandNewColumnBuilder.type(additionalColumn.getType());
+                }
+                if (additionalColumn.getName() != null) {
+                    brandNewColumnBuilder.name(additionalColumn.getName());
+                }
```
Comment (elogeart) [inline dataprep-actions/src/main/java/org/talend/dataprep/transformation/actions/common/ActionsUtils.java]: Is this change really necessary with the whole new code from the branch`jbrun/TDP-4926/bug_change_date_format_statistics` ? I am not sure about unwanted side-effects 🤔
